packages/lib/src/components/Dropin/Dropin.test.ts

```diff
@@ -217,4 +219,24 @@ describe('Dropin', () => {
             expect(dropin.props.openFirstStoredPaymentMethod).toBe(true);
         });
     });
+
+    describe('Detecting Enter key presses', () => {
+        test('should see merchant defined onEnterKeyPressed callback fired', done => {
+            new Dropin(checkout).mount('body');
```

Comment:
I think we could use the `render` utility function from the `@testing-library/preact` - `render(dropin.render());`. The reason is it cleans up the dom automatically, and more aligned with the rest of the tests.

Reply:
`render(dropin.render())` didn't work for me. The element, within the unit test, didn't fully render - so I could not access the element that I needed to, within the PM

Reply:
Maybe we need to wait for the pay button to be presented.

Reply:
I got it working it working with `render(dropin.render())` and `wiatfor` but then the event won't fire programatically!
If you want to refactor this test later be my guest!!